sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ThroughputControlGroupConfigBuilder.java

```diff
@@ -78,6 +80,19 @@ public ThroughputControlGroupConfigBuilder setDefault(boolean aDefault) {
         return this;
     }
 
+    /**
+     * Set whether allow request to continue on original request flow if throughput control controller failed on initialization.
+     * If set to true, requests will be able to fall back to original request flow if throughput control controller failed on initialization.
+     *
+     * @param continueOnInitError The flag to indicate whether request is allowed to fall back to original request flow.
+     * @return The {@link ThroughputControlGroupConfigBuilder}.
+     */
+    @Beta(value = Beta.SinceVersion.V4_28_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+    public ThroughputControlGroupConfigBuilder setContinueOnInitError(boolean continueOnInitError) {
```

Comment:
```suggestion
    public ThroughputControlGroupConfigBuilder continueOnInitError(boolean continueOnInitError) {
```

Reply:
I have been using set* pattern across in this class, for consistency consideration, should I change for all the methods?

Reply:
Discussed offline.
Will deprecate all previous set* public API
Add new public API which follows the builder pattern. Example, from setGroupName -> groupName